tests/accessor_basic/accessor_common.h

```diff
@@ -467,7 +463,6 @@ void check_empty_accessor_constructor_post_conditions(
   // All size queries return 0
   res_acc[res_i++] = testing_acc.byte_size() == 0;
   res_acc[res_i++] = testing_acc.size() == 0;
-  res_acc[res_i++] = testing_acc.max_size() == 0;
```

Comment:
What is the reason for this change?
NOTE: the code is removed for non-acpp compilers!